### PR TITLE
feat: Align test env with nova operator

### PR DESCRIPTION
**Pull request by bogdando**
* Switch testenv to ginkgo/v2
* Switch nova testenvs to lib-common test helper

### CODE REVIEW

**Comment by bogdando** [comment]
let's keep this open for a while, if you don't mind

File: go.mod [hunk 1]
```diff
@@ -14,13 +14,14 @@ require (
...
 	github.com/openstack-k8s-operators/mariadb-operator/api v0.0.0-20220927092716-25669bcdb523
 	github.com/openstack-k8s-operators/placement-operator/api v0.0.0-20220816132717-b1fe0c1140da
-	golang.org/x/mod v0.6.0
 	k8s.io/api v0.25.4
 	k8s.io/apimachinery v0.25.4
 	k8s.io/client-go v0.25.4
 	sigs.k8s.io/controller-runtime v0.13.1
 )
 
+require golang.org/x/mod v0.6.0 // indirect
```
Comment (gibizer) [inline go.mod:23]: this could go to the below require () block. I don't know why go mod put it separately.

**Review by gibizer — COMMENTED** [review]
Looks good to me so far. thanks

**Comment by bogdando** [comment]
PTAL

**Pull request merged**